collector-http/src/main/java/zipkin2/collector/otel/http/OpenTelemetryHttpCollector.java

```diff
@@ -206,11 +226,27 @@ protected HttpResponse doPost(ServiceRequestContext ctx, HttpRequest req) throws
           }
           collector.metrics.incrementBytes(content.length());
           try {
-            ExportLogsServiceRequest request = ExportLogsServiceRequest.parseFrom(
-                UnsafeByteOperations.unsafeWrap(content.byteBuf().nioBuffer()).newCodedInput());
+            MediaType contentType = msg.headers().contentType();
+            if (contentType == null) {
+              collector.metrics.incrementMessagesDropped();
+              return null;
+            }
+            ExportLogsServiceRequest request;
+            boolean base64Decoded = false;
+            if (contentType.isProtobuf()) {
+              request = ExportLogsServiceRequest.parseFrom(
+                  UnsafeByteOperations.unsafeWrap(content.byteBuf().nioBuffer()).newCodedInput());
```

Comment:
wonder if @ikhoon can take a look to see if this is the more efficient way to handle this (and the json block below)..